commit 5cc08ebf2f346992a0abd4440252165c90b5ec05
Author: Sergei Trofimovich <siarheit@google.com>
Date:   Sat Jun 20 12:23:00 2015 +0100

    Recognise 'hardhloat' as a valid vendor in a host tuple
    
    Observed on a tuple armv7a-hardfloat-linux-gnueabi:
    > Unknown vendor hardfloat
    
    Reported-by: Sergey Alirzaev
    Signed-off-by: Sergei Trofimovich <siarheit@google.com>

diff --git a/aclocal.m4 b/aclocal.m4
index 590edb0..958622c 100644
--- a/aclocal.m4
+++ b/aclocal.m4
@@ -1966,6 +1966,9 @@ AC_DEFUN([GHC_CONVERT_VENDOR],[
   softfloat) # like armv5tel-softfloat-linux-gnueabi
     $2="unknown"
     ;;
+  hardfloat) # like armv7a-hardfloat-linux-gnueabi
+    $2="unknown"
+    ;;
   *)
     #pass thru by default
     $2="$1"
